app/src/test/java/org/simple/clinic/patient/PatientMocker.kt

```diff
@@ -242,6 +242,7 @@ object PatientMocker {
       syncStatus: SyncStatus = SyncStatus.PENDING,
       agreedToVisit: Boolean? = null,
       remindOn: LocalDate? = LocalDate.now(UTC).minusDays(2),
+      isDefaulter: Boolean? = null,
```

Comment:
Let's use a random value so that `isDefaulter` is used in tests? 

```kotlin
isDefaulter: Boolean? = listOf(true, false, null).shuffled().first()
```

Reply:
I will write separate tests for checking random values of `isDefaulter` later.
Since I am not using this field at the moment.

Reply:
I'm sure you will, the randomness will be useful to other tests that are using `PatientMocker.appointment()`